acceptance/Admin.Specs/Hooks/AdminHooks.cs

```diff
@@ -0,0 +1,19 @@
+using TechTalk.SpecFlow;
+
+using Admin.Specs.Drivers;
+using Admin.Specs.PageObjects;
+
+namespace Admin.Specs.Hooks
+{
+
+    [Binding]
+    public class AdminHooks
+    {
+        [BeforeScenario("Admin")]
+        public static void BeforeScenario(BrowserDriver browserDriver)
+        {
+            var rootPageObject = new RootPageObject(browserDriver.Current);
+            // TODO clear login cookies
```

Comment:
Do we want to force a login in every scenario?

Reply:
With the fake OAuth service the cost of doing this seems to be very small ; with each test starting in a new browser profile this just seems to happen naturally anyway, I'll remove this comment because it's misleading (don't need to do this).